### PR TITLE
SC-19926 update sta removal instructions

### PR DESCRIPTION
**Pull request by cpipilas**
Update: https://sematext.com/docs/monitoring/spm-faq/#how-do-i-uninstall-the-sematext-agent for complete removal of Sematext Agent for various Linux distros.

### CODE REVIEW

**Comment by otisg** [comment]
I guess we'll add something for STAW? Or maybe https://sematext.com/docs/agents/sematext-agent/windows-installation/#how-to-unistall-sematext-agent will do?

File: docs/monitoring/spm-faq.md [hunk 1]
```diff
@@ -501,18 +501,117 @@ helm upgrade st-agent --set infraToken=<YOUR-INFRA-TOKEN> sematext/sematext-agen
...
+
+```bash
+sudo systemctl stop sematext-agent
+```
+
+#### 2. Remove the installation directory
+To make sure all configuration files and logs are removed, delete the main installation directory:
+
+```bash
+sudo rm -rf /opt/spm
```
Comment (boratanrikulu) [inline docs/monitoring/spm-faq.md:519]: after purge, not before.

File: docs/monitoring/spm-faq.md [hunk 2]
```diff
@@ -501,18 +501,117 @@ helm upgrade st-agent --set infraToken=<YOUR-INFRA-TOKEN> sematext/sematext-agen
...
-    monitored
+### How do I uninstall the Sematext Agent from my Linux System?
+
+If you need to uninstall and remove the `sematext-agent` package from your Linux system, follow the steps below for a clean and complete removal. The instructions cover various package managers commonly used in different Linux distributions.
+
+#### 1. Stop the sematext-agent Service (if running)
+First, make sure the `sematext-agent` service is stopped to avoid any running processes during the removal. Use the following command:
+
+```bash
+sudo systemctl stop sematext-agent
```
Comment (boratanrikulu) [inline docs/monitoring/spm-faq.md:512]: This shouldn't be needed.

**Review by boratanrikulu — COMMENTED** [review]
really hard to read, tab usage is needed. Suggest to move here https://sematext.com/docs/agents/sematext-agent/installation/ or create  uninstallation with tabs

File: docs/monitoring/spm-faq.md [hunk 3]
```diff
@@ -501,18 +501,117 @@ helm upgrade st-agent --set infraToken=<YOUR-INFRA-TOKEN> sematext/sematext-agen
...
+
+```bash
+sudo apt-get autoclean
+```
+
+##### For Red Hat-based systems
+Clean the metadata cache:
+
+```bash
+sudo yum clean metadata
```
Comment (boratanrikulu) [inline docs/monitoring/spm-faq.md:600]: too much?

File: docs/monitoring/spm-faq.md [hunk 4]
```diff
@@ -501,18 +501,117 @@ helm upgrade st-agent --set infraToken=<YOUR-INFRA-TOKEN> sematext/sematext-agen
...
+sudo zypper remove sematext-agent
+```
+
+#### 4. Clean the package cache
+To free up disk space and remove package files that are no longer needed, use the appropriate command for your package manager:
+
+##### For Debian-based systems
+
+```bash
+sudo apt-get clean
```
Comment (boratanrikulu) [inline docs/monitoring/spm-faq.md:566]: can be removed

**Pull request closed without merging**